pkg/minikube/bootstrapper/bsutil/kverify/kubelet.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Since this file is about services, we should probably rename it from `kubelet.go`. I also think these functions would make more sense in the systemd package for that reason

Reply:
thanks will move to a different file

Reply:
make sense, changed file name but moving that functionality to systemd package, it will be a follow up refactor PR. since it does not concern adding new component--wait flag in this PR